### PR TITLE
Story1 task1

### PR DESCRIPTION
**Pull request by SamBellDev**
HTML concept frame

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    <input type="submit" name="submit">
+</form>
+<p> Item 1
+    <ul>
+        <li>stat 1
+        </li>
+        <li>stat 2
+        </li>
+        <li>stat 3
+        </li>
```
Comment (MrLukey) [inline index.php:37]: list items can be on the same line e.g `<li>content</li>`

File: index.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    <meta charset="utf-8">
+</head>
+
+<body>
+<h1>My collection</h1>
+<form>
+    <label>
+        New Item:
+        <input type="text" name="newItem">
+    </label>
```
Comment (MrLukey) [inline index.php:15]: Inputs are inside the label rather than next to. Also `for='input-id'` will link the inputs and labels

**Review by MrLukey — CHANGES_REQUESTED** [review]

File: index.php (hunk 1 above)
Comment (SamBellDev) [reply index.php]: restructured.

**Review by MrLukey — APPROVED** [review]
All gravy baby

**Pull request merged**